geoviews/element/geo.py

```diff
@@ -311,20 +303,18 @@ class Dataset(_Element, HvDataset):
 
 
 class Points(_Element, HvPoints):
-    """
-    Points represent a collection of points with
+    """Points represent a collection of points with
     an associated cartopy coordinate-reference system.
     """
 
     group = param.String(default='Points')
 
     def geom(self, union=False, projection=None):
-        """
-        Converts the Points to a shapely geometry.
+        """Converts the Points to a shapely geometry.
 
         Parameters
         ----------
-        union: boolean (default=False)
+        union: boolean, default=False
```

Comment:
There must be a space between the parameter name and the colon (https://numpydoc.readthedocs.io/en/latest/format.html#parameters) (I haven't checked the whole PR, just spotted this one).

```suggestion
        union : boolean, default=False
```